.buildkite/docker/build-and-run-load.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+set -e
+set -x
+
+node --version
+npm --version
+
+npm set progress=false
+export SKIP_DOCS_INSTALL=true
+export TEMPORAL_TESTING_SERVER_URL="temporal:7233"
+
+npm ci
+# Try rebuilding once if it fails
+# TODO: Figure out whatever problem causes "common" package to not get compiled when it should
+npm run build || npm run build
```

Comment:
Wow, this is how you solved the build problem?
Let's keep the hack and track the issue.